README.md

```diff
@@ -1,39 +1,30 @@
-# PG2: A *Fast* PostgreSQL Driver For Clojure
```

Comment:
Why did you remove A? It relates to the "Driver" noun which is mentioned the first time.